### ISSUE TITLE
Using bento command-line under cygwin. 

### ISSUE THREAD

**Issue by OliDrouin** [chef/bento#1262]
# Version:

master

# Environment:

cygwin

# Scenario:

Trying to use "bento" on command line (https://github.com/chef/bento/wiki/Using-the-Bento-binary-to-build---publish-boxes) under cygwin fails. 

# Steps to Reproduce:
```
t898379@L103363 /cygdrive/c/dev/bento
$ bundle exec $DEV/bento/bin/bento build --only=virtualbox-iso  packer_templates/ubuntu/ubuntu-18.04-amd64.json
==> Starting build for templates:
- packer_templates/ubuntu/ubuntu-18.04-amd64
==> [ubuntu-18.04-amd64] Building: 'packer build -var headless=true -only=virtualbox-iso -force -var-file=/tmp/ubuntu-18.04-amd64-metadata.json20191213-40543-7u83ly ubuntu-18.04-amd64.json'
invalid value "/tmp/ubuntu-18.04-amd64-metadata.json20191213-40543-7u83ly" for flag -var-file: open /tmp/ubuntu-18.04-amd64-metadata.json20191213-40543-7u83ly: The system cannot find the path specified.
Usage: packer build [options] TEMPLATE

  Will execute multiple builds in parallel as defined in the template.
  The various artifacts created by the template will be outputted.

Options:

  -color=false                  Disable color output. (Default: color)
  -debug                        Debug mode enabled for builds.
  -except=foo,bar,baz           Run all builds and post-procesors other than these.
  -only=foo,bar,baz             Build only the specified builds.
  -force                        Force a build to continue if artifacts exist, deletes existing artifacts.
  -machine-readable             Produce machine-readable output.
  -on-error=[cleanup|abort|ask] If the build fails do: clean up (default), abort, or ask.
  -parallel=false               Disable parallelization. (Default: true)
  -parallel-builds=1            Number of builds to run in parallel. 0 means no limit (Default: 0)
  -timestamp-ui                 Enable prefixing of each ui output with an RFC3339 timestamp.
  -var 'key=value'              Variable for templates, can be used multiple times.
  -var-file=path                JSON file containing user variables.
>>> [ubuntu-18.04-amd64] Error building, exited

```

# Expected Result:

bento builds the box so that I can "upload" and then "publish" under my vagrant cloud account. 

# Actual Result:

command fails with 

```
==> [ubuntu-18.04-amd64] Building: 'packer build -var headless=true -only=virtualbox-iso -force -var-file=/tmp/ubuntu-18.04-amd64-metadata.json20191213-40543-7u83ly ubuntu-18.04-amd64.json'
invalid value "/tmp/ubuntu-18.04-amd64-metadata.json20191213-40543-7u83ly" for flag -var-file: open /tmp/ubuntu-18.04-amd64-metadata.json20191213-40543-7u83ly: The system cannot find the path specified.
Usage: packer build [options] TEMPLATE

```

**Comment by tas50** [comment]
Thanks for opening this issue. I ran into this today actually and I was going to get an issue opened. The bento experience isn't fully functional on windows yet and that needs to get fixed so you can fully upload hyper-v or other boxes.

**Comment by arizvisa** [comment]
@OliDrouin, you might be able to influence the temp-directory that is used by setting the environment variables that ruby uses for `Dir::tmpdir` via assigning a windowsy path to `TMP`, `TMPDIR`, or `TEMP`.

**Comment by Stromweld** [comment]
closing this issue due to age. Please try the latest bento release in native windows, wsl, or cygwin and reopen or submit a new issue if this is still a problem.
